packages/p2p-v2/src/pages/my-ads/components/AdStatus/AdStatus.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Text } from '@deriv-com/ui';
+import { useDevice } from '../../../../hooks';
+import './AdStatus.scss';
+
+type TAdStatusProps = {
+    isActive?: boolean;
+};
+
+const AdStatus = ({ isActive = false }: TAdStatusProps) => {
+    const { isMobile } = useDevice();
+    if (!isActive) {
+        return (
+            <Text
+                align='center'
+                className='p2p-v2-ad-status--inactive'
+                color='error'
+                size={isMobile ? 'md' : 'sm'}
+                weight='bold'
+            >
+                Inactive
+            </Text>
+        );
+    }
+
+    return (
+        <Text
+            align='center'
+            className='p2p-v2-ad-status--active'
+            color='success'
+            size={isMobile ? 'md' : 'sm'}
+            weight='bold'
+        >
+            Active
+        </Text>
+    );
+};
```

Comment:
can we do something like this

```suggestion
    return (
        <Text
            align='center'
            className={clsx({
                  'p2p-v2-ad-status--active': isActive,
                  'p2p-v2-ad-status--inactive': !isActive,
            )}
            color={isActive ? 'success' : 'error'}
            size={isMobile ? 'md' : 'sm'}
            weight='bold'
        >
            { isActive ? 'Active' : 'Inactive' }
        </Text>
    );
};
```